pom.xml

```diff
@@ -520,12 +149,12 @@
         <repository>
             <id>nexus-releases</id>
             <name>WSO2 Release Distribution Repository</name>
-            <url>http://maven.wso2.org/nexus/service/local/staging/deploy/maven2/</url>
+            <url>https://maven.wso2.org/nexus/service/local/staging/deploy/maven2/</url>
```

Comment:
Shall we have the direct prod url here? Similar to https://github.com/wso2-extensions/esb-connector-kafka/blob/master/pom.xml#L420